v3.3/glfw/native_linbsd_wayland.go

```diff
@@ -0,0 +1,47 @@
+// +build linux,wayland freebsd,wayland netbsd,wayland
```

Comment:
Shouldn't this contain `openbsd,wayland` as well?

Reply:
yes, updated

Reply:
Thanks.

Reply:
Now that the new file has `openbsd,wayland` added, it seems `native_linbsd_x11.go` should also be updated so its build constraint is `openbsd,!wayland`.

Reply:
Good catch. That must have been missed when the OpenBSD support was added in.